### PR TITLE
[DO NOT MERGE]Try to set up a OSD cluster

### PR DESCRIPTION
**Pull request by huweihua-redhat**
Try to set up a OSD cluster

### CODE REVIEW

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

File: ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ ocm login --url "${OCM_LOGIN_ENV}" --token "${OCM_TOKEN}"
 CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
 echo "Deleting cluster: ${CLUSTER_ID}"
 
+sleep 86400
```
Comment (xueli181114) [inline ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh:19]: For the time sleep, you want to keep the cluster for 1 day rignt?

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

File: ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh (hunk 1 above)
Comment (huweihua-redhat) [reply ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh:19]: yes.  I just want to set up a OSD cluster,  this PR won't be merged. thanks a lot

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-osd-ocp413-quay-e2e-tests-quay38-osd-ocp413

**Pull request closed without merging**